src/bb_spi_lcd.cpp

```diff
@@ -4327,12 +4330,6 @@ uint16_t *u16Temp = (uint16_t *)ucRXBuf;
         return 0;
     }
     spilcdSetPosition(pLCD, 0,0,pLCD->iCurrentWidth,pLCD->iCurrentHeight, iFlags);
-    // fit within our temp buffer
-    cx = 1; tx = pLCD->iCurrentWidth;
-    if (pLCD->iCurrentWidth > 160)
-    {
-       cx = 2; tx = pLCD->iCurrentWidth/2;
-    }
```

Comment:
These lines are entirely unused in both cases below:
- `#ifdef __AVR__`: both `cx` and `tx` are immediately assigned to, discarding any work done by these lines.
- `#else`: `cx` and `tx` are unreferenced, ignoring any work done by these lines. The comment in the PR description suggests they were intended for use in this block, but it wasn't clear to me how.